src/main/java/com/opentable/db/postgres/embedded/PreparedDbProvider.java

```diff
@@ -216,5 +236,21 @@ private static void create(final DataSource connectDb, final String dbName, fina
             this.user = null;
             this.ex = e;
         }
+
+        public int getPort() {
+            return port;
+        }
+
+        public String getDbName() {
+            return dbName;
+        }
+
+        public String getUser() {
+            return user;
+        }
+
+        public SQLException getEx() {
+            return ex;
```

Comment:
If we're going to make this public, how about we name it more explicitly?

Reply:
Ok.